src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -230,6 +277,7 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
         }
 
         // Check if the motion between the nearest state and the state to add is valid
+        ++collisionChecks_;
```

Comment:
`SpaceInformation::checkMotion` can correspond to several collision checks. If you want the number of collision checks, you'd need to count the number of calls to the various StateValidityChecker::isValid(...) methods. It's easy enough to keep track of this with an internal counter in the StateValidityChecker, but I'm not sure we should make this a requirement for a StateValidityChecker-derived class. But suppose you do this, then you can update ompl::base::Planner::getPlannerData() to add the number of collision checks as an extra property.